js/src/tooltip.js

```diff
@@ -71,7 +69,7 @@ const AttachmentMap = {
 
 const Default = {
   animation: true,
-  template: '<div class="tooltip" role="tooltip">' +
+  template: '<div class="tooltip bs-tooltip" role="tooltip">' +
```

Comment:
Instead of adding this `.bs-tooltip`  class, just `.tooltip` can not  be used here :thinking:

Reply:
@mdo Can we have a little help on this? I am not sure how to handle the scss file. It still has the `bs-tooltip-{position}` classes, which the `[data-popper-placement^="top"]` extends

May we use it with the  extra `bs-tooltip-auto` class, or change the scss, so we end up using only the `tolltip` class?

Feel free to commit any change